Hazelcast.Net/Hazelcast.Config/ClientNetworkConfig.cs

```diff
@@ -91,36 +103,66 @@ public virtual IList<string> GetAddresses()
             return _addressList;
         }
 
+        /// <summary>
+        /// While client is trying to connect initially to one of the members in the configured address list,
+        /// all might be not available. Instead of giving up, throwing Exception and stopping client, it will
+        /// attempt to retry as much as connection Attempt Limit times.
+        /// </summary>
+        /// <returns>number of times to attempt to connect</returns>
         public virtual int GetConnectionAttemptLimit()
         {
             return _connectionAttemptLimit;
         }
 
+        /// <summary>
+        /// Returns connection attempt period in millis, Period for the next attempt to find a member to connect.
```

Comment:
/// Returns the connection attempt period, in millis, for the next attempt to find a member to connect.